ansible-st2/roles/st2/templates/st2reactor/st2sensorcontainer.j2

```diff
@@ -1,24 +0,0 @@
-description     "StackStorm st2sensorcontainer service"
```

Comment:
#51 Check what these files look like on live system, and which service reads them.